### PR TITLE
Version major increment for PR#123

### PR DESCRIPTION
**Pull request by kkudose**
[PR #123](https://github.com/ShipEngine/connect/pull/123) needed a version increment

### CODE REVIEW

**Comment by kkudose** [comment]
increment major

File: packages/carrier-api/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shipengine/connect-carrier-api",
-  "version": "2.0.0",
+  "version": "3.0.0",
```
Comment (lzhang) [inline packages/carrier-api/package.json:3]: Do we actually need to bump versions for this? Maybe we can just let the next functional change bump it.

Comment (kkudose) [reply packages/carrier-api/package.json:3]: The version change is actually for [this PR that removes deprecated fields](https://github.com/ShipEngine/connect/pull/123). I've updated this PR to be more clear as well (we weren't sure if this was going to work)

File: packages/connect/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shipengine/connect",
-  "version": "1.12.0",
+  "version": "2.0.0",
```
Comment (ricky) [inline packages/connect/package.json:3]:
```suggestion
  "version": "1.12.1",
```

Although this _is_ a breaking change in the api & runtime, it doesn't actually change the behavior from the perspective of the app developer, so we don't need a major bump here.

Comment (kkudose) [reply packages/connect/package.json]: Updated to 1.12.2 (PR #124 updated to 1.12.1)

**Comment by mattburton** [comment]
Closing - reverting in #132

**Pull request closed without merging**